.github/workflows/tests.yml

```diff
@@ -27,6 +27,8 @@ jobs:
                       sf-version: '6.0.*'
                     - php-version: '8.1'
                       sf-version: '6.0.*'
+                    - php-version: '8.1'
+                      sf-version: '6.1.*'
```

Comment:
Do you think we can reduce it to just: ?

```yml
php-version: ['7.4', '8.0', '8.1']
sf-version: ['4.4.*', '5.4.*', '6.0.*', '6.1.*']
```

Reply:
We cannot go this way as Symfony 6.1 only supports php ^8.1 :)